expanse.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=web app for storing Reddit items
+Documentation=https://github.com/jc9108/expanse
+Wants=network-online.target
+Requires=docker.service
+After=network-online.target docker.service
+
+[Service]
+Type=exec
+WorkingDirectory=/opt/expanse
+ExecStartPre=-run.sh prod update
+ExecStart=run.sh prod up --no-daemonize
```

```suggestion
ExecStart=sh ./run.sh prod up --no-d
```
add `sh`, use relative path